GUI/window.py

```diff
@@ -0,0 +1,89 @@
+
+from kivy.app import App
+from kivy.uix.boxlayout import BoxLayout
+from kivy.uix.recycleview.views import RecycleDataViewBehavior
+from kivy.uix.button import Button
+from kivy.properties import BooleanProperty, ListProperty, StringProperty, ObjectProperty
+from kivy.uix.recyclegridlayout import RecycleGridLayout
+from kivy.uix.behaviors import FocusBehavior
+from kivy.uix.recycleview.layout import LayoutSelectionBehavior
+from kivy.uix.popup import Popup
+from kivy.config import Config
+Config.set('graphics', 'width', '1024')
+Config.set('graphics', 'height', '650')
+Config.set('graphics', 'resizable', False)
+
+# Window tables with editable data rows
+#Uses test.kv as a config file
+
+class TextInputPopup(Popup):
+    obj = ObjectProperty(None)
+    obj_text = StringProperty("")
+
+    def __init__(self, obj, **kwargs):
+        super(TextInputPopup, self).__init__(**kwargs)

```

Comment:
Why do you not pass obj to super?

Reply:
Should I? I thought that by passing the class `TextInputPopup` was enough.

Reply:
Does the super class also have `obj` or is that something you defined?

Reply:
It's something I defined, and I'm just initializing it under the super passing. This obj is used to read the content when editing the rows of data on the window. Technically it is not needed, but it is a nice feature to know about to modify it later on.